spec/factories/ont/libraries.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :ont_library, class: 'Ont::Library' do
+    plate_barcode { "PLATE-1-123456" }
+    pool { 2 }
+    well_range { "A1-H8" }
```

Comment:
Aren't the wells `A1-H12` on these plates?

Reply:
As this is a test factory, I'm not going to change this. I think the data added here is sufficient for testing purposes